client/api/src/main/java/org/opennms/velocloud/client/api/internal/Utils.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+package org.opennms.velocloud.client.api.internal;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+public class Utils {
+    public static InetAddress getValidInet4Address(String ip) {
+        try {
+            InetAddress address = InetAddress.getByName(ip);
+            if (address.getHostAddress().equals(ip)) {
+                return address;
+            } else return null;
```

Comment:
Consider using this instead: https://guava.dev/releases/19.0/api/docs/com/google/common/net/InetAddresses.html#forString(java.lang.String)